components/speedreader/rust/lib/src/readability/src/extractor.rs

```diff
@@ -439,30 +485,84 @@ mod tests {
     fn test_clean_title_colon() {
         let input = "The SoCal Weekly Digest: Welcome to our wonderful page";
         let expected = "Welcome to our wonderful page";
-        let output = clean_title(input.to_string());
+        let output = clean_title(&Sink::default(), input.to_string());
         assert_eq!(expected, output);
     }
 
     #[test]
     fn test_clean_title_separator_left() {
         let input = "Príncipe Harry asegura que su padre y hermano están \"atrapados\" en la monarquía: \"Siento compasión\" | Príncipe Carlos | Príncipe William | Meghan Markle | Duques de Sussex | Oprah Winfrey";
         let expected = "Príncipe Harry asegura que su padre y hermano están \"atrapados\" en la monarquía: \"Siento compasión\"";
-        let output = clean_title(input.to_string());
+        let output = clean_title(&Sink::default(), input.to_string());
         assert_eq!(expected, output);
     }
 
     #[test]
     fn test_clean_title_separator_right() {
         let input = "Short Title | How Cats Can Save the Planet";
         let expected = "How Cats Can Save the Planet";
-        let output = clean_title(input.to_string());
+        let output = clean_title(&Sink::default(), input.to_string());
         assert_eq!(expected, output);
     }
 
     #[test]
     fn test_clean_title_preserve_hyphen() {
         let input = "Just-released Minecraft exploit makes it easy to crash game servers";
-        let output = clean_title(input.to_string());
+        let output = clean_title(&Sink::default(), input.to_string());
+        assert_eq!(input, output);
+    }
+
+    #[test]
+    fn test_clean_title_ascii_dash_separator() {
+        // A common pattern found it <title> tags is the site name being included after a final dash
+        let input =
+            "House committee votes to approve bill that would grant DC statehood - CNNPolitics";
+        let expected = "House committee votes to approve bill that would grant DC statehood";
+        let output = clean_title(&Sink::default(), input.to_string());
+        assert_eq!(expected, output);
+    }
+
+    #[test]
+    fn test_clean_title_unicode_dash_separator() {
+        // A follow up to the last test with common unicode dashes. For example &#8211; converts to an en dash.
+        let dashes = [
+            "-",   // hyphen
```

Comment:
nit: extra space in the comment

Reply:
i agree but for whatever reason `rustfmt` is doing it